src/DataTable/TableFilters.jsx

```diff
@@ -2,13 +2,21 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Button from '../Button';
 
+export const TABLE_FILTERS_BUTTON_TEXT = 'Filter';
+
 const TableFilters = ({
   columns, manualFilters, onFilter, currentFilters,
 }) => (
   <div>
     <h4>Filters</h4>
     {columns.map(column => <div key={column.Header}>{column.canFilter ? column.render('Filter') : null}</div>)}
-    {manualFilters && <Button type="primary" onClick={() => onFilter(currentFilters)}>Filter</Button>}
+    {manualFilters && (
+    <Button
+      type="primary"
+      onClick={() => onFilter(currentFilters)}
+    >{TABLE_FILTERS_BUTTON_TEXT}
+    </Button>
```

Comment:
Please, fix the code formating
```suggestion
    <Button
      type="primary"
      onClick={() => onFilter(currentFilters)}
    >
        {TABLE_FILTERS_BUTTON_TEXT}
    </Button>
```

Reply:
Done